VideoAPI/VideoApi.Common/Security/Kinly/KinlyConfiguration.cs

```diff
@@ -7,5 +7,6 @@ public class KinlyConfiguration
         public string Audience { get; set; }
         public string Issuer { get; set; }
         public string CallbackSecret { get; set; }
+        public string ConferencePhoneNumber { get; set; }
```

Comment:
Could these 2 have the same name for consistency?
ConferencePhoneNumber 
and
TelephoneConferenceNumber

Reply:
I stuck with the name in the config as done in admin-web and in the library.
The name in the response model is more consistent